app/components/recirculation-prefooter.js

```diff
@@ -58,23 +67,34 @@ export default Component.extend(
       });
 
       this.ads.addWaitFor('RECIRCULATION_PREFOOTER', this.get('listRendered.promise'));
+
+      blackList.clear();
```

Comment:
Can we import it as recirculationBlacklist so that it's clear what the blacklist is for?